lib/rules/web/ocm_console/cluster_list_page.xyaml

```diff
@@ -414,6 +414,7 @@ logout_page_loaded:
 go_to_osd_cluster_creation_page:
```

Comment:
We have a function switch_to_osd_creation_page.

Reply:
updated,thanks.